contents/docs/deploy.mdx

```diff
@@ -0,0 +1,74 @@
+---
+title: Deploy PostHog
+sidebar: Docs
+---
+
+PostHog offers a number of different deployment options:
+
+-   [PostHog Cloud](#posthog-cloud) - Create a new PostHog Cloud deployment with just a few clicks.
+    -   Recommended as it's **the fastest way** to try PostHog out.
+    -   Comes with **extra features for reliability and security** and automatic updates.
+-   [Hobby](#hobby-deployment) - Recommended for testing or very small hobby projects _(&lt;100k events/month)_
+-   [PostHog Self-hosted](#posthog-self-hosted) - Recommended for most self-host deployments.
```

Comment:
Self-hosted is recommended for people who need to self-host doesn't really add much. I'd suggest changing it to...

```suggestion
-   [PostHog Self-hosted](#posthog-self-hosted) - Recommended for users who need to keep data on their own infrastructure or private cloud.
```